<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductDialog</class>
 <widget class="QDialog" name="AddProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Product</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgs.qrc">
    <normaloff>:/images/icons8-new-48.png</normaloff>:/images/icons8-new-48.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Code :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codeLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Price : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="priceLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Production Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="productiondateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Expiration Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="expirationDateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Quantity : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="quantityLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Inventory : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imgs.qrc"/>
 </resources>
 <connections/>
</ui>
